projects/packages/forms/src/dashboard/class-dashboard.php

```diff
@@ -112,13 +112,12 @@ public function load_admin_scripts() {
 		// Adds Connection package initial state.
 		Connection_Initial_State::render_script( self::SCRIPT_HANDLE );
 
-		$api_root = defined( 'IS_WPCOM' ) && IS_WPCOM
-			? sprintf( '/wpcom/v2/sites/%s/', esc_url_raw( rest_url() ) )
-			: '/wp-json/wpcom/v2/';
-
+		// Preload Forms config endpoint for dashboard context.
+		$preload_paths = array( '/wp/v2/feedback/config' );
+		$preload_data  = array_reduce( $preload_paths, 'rest_preload_api_request', array() );
 		wp_add_inline_script(
 			self::SCRIPT_HANDLE,
-			'window.jetpackFormsData = ' . wp_json_encode( array( 'apiRoot' => $api_root ) ) . ';',
+			'wp.apiFetch.use( wp.apiFetch.createPreloadingMiddleware( ' . wp_json_encode( $preload_data ) . ' ) );',
```

Comment:
The 'block_editor_rest_api_preload' filter does not run on non-editor pages by default. As I understand it, the code above is how you leverage this on normal admin pages. This is relatively novel, and it would be good to get a second opinion on this.

Reply:
Nice. 

I think in future requests it would be good to also load other endpoints but we can do this in different PRs. :)